src/pages/profiles/ProfileStorageList.tsx

```diff
@@ -2,26 +2,27 @@ import { FC } from "react";
 import { LxdProfile } from "types/profile";
 import { isDiskDevice } from "util/devices";
 import ExpandableList from "components/ExpandableList";
+import ResourceLink from "components/ResourceLink";
 
 interface Props {
   profile: LxdProfile;
+  project: string;
 }
 
-const ProfileStorageList: FC<Props> = ({ profile }) => {
+const ProfileStorageList: FC<Props> = ({ profile, project }) => {
   return (
     <>
       {Object.values(profile.devices).some(isDiskDevice) ? (
         <ExpandableList
           items={Object.values(profile.devices)
             .filter(isDiskDevice)
             .map((device) => (
-              <div
+              <ResourceLink
                 key={device.path}
-                className="u-truncate list-item"
-                title={`Pool ${device.pool}`}
-              >
-                {device.pool}
-              </div>
+                type="pool"
+                value={device.pool}
+                to={`/ui/project/${project}/storage/pool/${device.pool}`}
+              />
```

Comment:
There is a bug here, when assigning storage volumes to a profile, they will appear as storage pool.
I think the right solution is to use the same approach as on the instance overview page. So remove the storage list, and introduce a device list here.

Maybe this can be done with a shared component. If we can share code, it would be great.

This is a bit out of scope for the current task, might be better to create a task and follow up pr for this problem.

Reply:
Yes, I think let's keep this outside the scope of current PR. At the moment both volume and pool has the same icon so it's still okay, but will create a task for this.

Reply:
Yeah was just doing it, here's the [issue](https://warthogs.atlassian.net/browse/WD-16007?atlOrigin=eyJpIjoiMjdmN2MwYzBlZTUzNDJlM2E0NjIwODJiNWY2Zjk3ZDQiLCJwIjoiaiJ9)